Christmas Updates

arpitkarsolia opened the pull request.

Commit:
Merge pull request #5 from chalmers-revere/master

stable

Commit:
* christmas updates

Commit:
*version updates

Commit:
* new Rhino CAN interface

Commit:
* remove old Rhino CAN microservice

Commit:
* minor updates

Commit:
* deleting obsolete file

chrberger committed:
Bumping microservice for OxTS to v0.0.16

Commit:
Merge pull request #7 from chalmers-revere/2019Q1.feature.oxts-filter

Bumping microservice for OxTS to v0.0.16

chrberger reviewed (approved):
LGTM

chrberger commented:
We should test this release and make a new PR to have VV 0.0.59

Commit:
* updated VV to v0.0.59

chrberger reviewed (commented):
Before we roll out v0.0.59, we should run one more test on the car.

Commit:
*major updates

- OxTS version update
- VLP32C GPS time added
- 2 axis cameras (new microservice)
- CPU Priority
- separate CID's 
- Downsampling

Commit:
'renaming from rhino.yml to GoldenPC.yml

Commit:
*indentation error - CAN

chrberger committed:
Upgrading to opendlv-vehicle-view v0.0.60

chrberger committed:
dos2unix

Signed-off-by: Christian Berger <christian.berger@gu.se>

chrberger committed:
Intel QuickSync for Rhino (Axis Out)

Signed-off-by: Christian Berger <christian.berger@gu.se>

chrberger committed:
Update GoldenPC.yml

* fixed missing executable for CANgw
* commented out video interface
* added thumbnail (commented) for Axis In

Commit:
updates & reverting CID=100

Signed-off-by: Arpit Karsolia <arpit.karsolia@chalmers.se>

Commit:
*cosmetic update

Commit:
* oxts ip update

Commit:
* oxts ip update

Commit:
*network arch

Commit:
* network arch rhino
